app/views/event_emails/new.html.haml

```diff
@@ -21,7 +21,7 @@
   = simple_form_for(resource, url: event_event_emails_path) do |f|
     = f.input :to, as: :text,readonly: true, input_html: {rows: 1}
     = f.input :subject
-    = f.input :body, as: :text, input_html: {rows: 12}
+    = f.input :body, as: :text, hint: 'This field supports markdown', input_html: {rows: 12}
```

Comment:
Please add a link to a markdown cheat sheet
e.g.
https://daringfireball.net/projects/markdown/syntax